grant/projects/sublayer.md

```diff
@@ -60,3 +62,32 @@ SubLayer will also develop advanced trading features like biding, rewarding, dai
 SubLayer phase 1 development work was kicked off after CNY 2024. Phase 1 is expected to finish before end of March 2024.
 
 As SubLayer team believes, all blockchain ecosystem will merge into the same metaverse, so we build to make this come true.
+
+### Delivery
+
+#### Phase 1
+
+SubLayer phase 1 onboarded Darwinia Koi Testnet in Sep and Mainnet in Nov 2024.
+
+The delivery includes:
+
+- Smart contract(https://github.com/sublayer-art/sublayer-protocol)
+    - ERC721 NFT exchange using order book mechanism
+    - ERC721 NFT template with enumerable and uri storage extension
+
+- Web repo (https://github.com/sublayer-art/sublayer-web)
+    - Trending NFT collection list
+    - NFT collection launchpad
+    - NFT marketplace supporting buy and list
+    - User NFT portfolio
+    - Data analysis on minting(count) and trading(count and price)
+    - Code is written in React + TypeScript + Vite framework
+
+- Server repo (https://github.com/sublayer-art/sublayer-api)
+    - NFT events indexing (Registered sublayer collections)
+    - NFT collection, minting, trading, analyzing API
+    - API document included in the code using swagger
+
+### Conclusion
+
+The Sublayer project has successfully completed the first-stage demand for NFT trading market in Ring DAO ecosystem, and through interactions with the community, we also got good feedback. We are delighted to receive $4,000 support from Ring DAO for Grant. Next, Sublayer will continue to improve and strive to become the leading NFT trading markets in both Darwinia and Polka ecosystems.
```

Comment:
As discussed with Bear and Xin from Sublayer, $4K will be issued as a reward for this partial delivery, and this grant will be closed after the reward for the partial delivery is issued. Any subsequent proposals will require a new application and grant submission.